packages/internal/src/build/web.ts

```diff
@@ -35,8 +35,16 @@ export const prebuildWebFiles = (srcFiles: string[], flags?: Flags) => {
       return undefined
     }
 
+    // Fixes nodeFileTrace errors on windows when building
```

Comment:
I don't think we use NFT in this function - in fact this file isn't actually used in the build for the web side. Would it be ok to remove?

Reply:
@dac09 
Sorry, I should say Zipping. and you are right we don't zip the webside. I just checked to see if babel was having an issue with the webside windows pathing and it does not. Both with and without, the files are identical down to the byte so its not needed for now. My question is should I remove it all, or just comment out the lines for future reference for possible future needs?

Reply:
I think its best not to have any of these changes in the build/web or build/api files, since it specifically applies to deploying serverless, so should only be in the code that involves deploying with sls :)

Reply:
I agree this feels like a temporary patch, I think it would be neat for `nodeFileTrace `to support wack windows pathing. but we use `nodeFileTrace `on the built files. So I'm seeing alternate solutions such as either we make babel output to `/winDist/` folder that gets created only when on windows systems that then applies the pathing fix, or call `nodeFileTrace` on the `/src/` files but that exposes us to many unknown errors with different IDEs.

Reply:
the other conversation is getting somewhere ill hit the resolve conversation on this chain. and start thinking of this as more of a temporary patch.